src/Compilers/Core/Portable/CaseInsensitiveComparison.cs

```diff
@@ -183,10 +183,7 @@ public override int GetHashCode(string str)
         /// <summary>
         /// Returns a StringComparer that compares strings according the VB identifier comparison rules.
         /// </summary>
-        public static StringComparer Comparer
-        {
-            get { return s_comparer; }
-        }
+        public static StringComparer Comparer => s_comparer;
```

Comment:
That syntax will be used in the presence of primary constructors to initialize a compiler-generated backing field at construction-time for a getter-only property.